specs/engine.spec/src/main/resources/META-INF/zilla/core.idl

```diff
@@ -119,12 +119,12 @@ scope core
 
         struct BindingEvent extends Event
         {
-            int64 routedId = 0;
-            int64 initialId = 0;
+            int64 bindingId = 0;
         }
 
         struct CatalogEvent extends Event
         {
+            string8 catalogId = null;
```

Comment:
This is `int64` similar to `bindingId` above.

Reply:
fixed